### PR TITLE
Fixes an issue with flaky 262 test

### LINKED ISSUE #4405: Flaky test262/error on release/1.7 (Windows)
```
07:54:32 3>    ch.exe    -DumpOnCrash -WERExceptionSupport -ExtendedErrorStackForTestHost -BaselineMode -forceNative -off:simpleJit -bgJitDelay:0 -dynamicprofileinput:profile.dpl.UnnamedTest62  -$262 D:\j\w\ci_dev12_x64_---276a6eb6\test\$262\$262test.js >D:\j\w\ci_dev12_x64_---276a6eb6\test\$262\testout2 2>&1
07:54:32 3>ERROR: name of output file: D:\j\w\ci_dev12_x64_---276a6eb6\test\$262\testout2; size: 152; creation: Fri Dec 08 07:45:55 2017, last access: Fri Dec 08 07:45:55 2017, now: Fri Dec 08 07:45:57 2017
07:54:32 3>ERROR: bad output file follows ============
07:54:32 3>test262/error Test262Error: Expected SameValue(«1ok», «0ok») to be true
07:54:32 3>test262/error Test262Error: Expected SameValue(«0ok», «1ok») to be true
```

Not sure whom this should be assigned. If no-one looks at, I will take a look after break.

### PR DESCRIPTION
**Pull request by obastemur**
fixes #4405

### CODE REVIEW

File: bin/ch/WScriptJsrt.cpp [hunk 1]
```diff
@@ -947,18 +947,16 @@ bool WScriptJsrt::Initialize()
         IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Leaving", LeavingCallback));
         IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Sleep", SleepCallback));
 
-
-        // OSX does build does not support $262 as filename
-        const wchar_t $262[] =
+        const char Test262[] =
```
Comment (jackhorton) [inline bin/ch/WScriptJsrt.cpp]: While we are changing code here, can we add a comment here and/or in 262.js that explains this code pattern?

File: test/262/262test.js [hunk 2]
```diff
@@ -78,7 +77,7 @@ assert.notSameValue = function (actual, unexpected, message) {
     message += ' ';
   }
 
-  message += 'Expected SameValue(«' + String(actual) + '», «' + String(unexpected) + '») to be false';
+  message += 'Expected SameValue(<' + String(actual) + '>, <' + String(unexpected) + '>) to be false';
```
Comment (jackhorton) [inline test/262/262test.js]: The benefit of keeping the unicode angle brackets is that that is what test262 prints itself (like when run through [test262-harness](https://github.com/bterlson/test262-harness))

File: bin/ch/WScriptJsrt.cpp (hunk 1 above)
Comment (obastemur) [reply bin/ch/WScriptJsrt.cpp]: Good idea.

**Comment by obastemur** [comment]
@leirocks could you please review

**Comment by obastemur** [comment]
@jackhorton  @leirocks thanks for the review!

**Pull request merged**